snapraid-runner.py

```diff
@@ -106,6 +106,10 @@ def send_email(success):
         server = smtplib.SMTP_SSL(**smtp)
     else:
         server = smtplib.SMTP(**smtp)
+        server.ehlo()
```

Comment:
what happens if this is sent to a server that doesn't understand `EHLO`?

Reply:
EHLO is specified in the RFC 2821 from 2001, I would expect that 16 years later it is implemented in every SMTP server. I didn't find any modern SMTP server that does not have EHLO implemented.

Reply:
That is a very hopeful attitude towards internet standards :). The IPv6 standard was released in 1998.
I looked it up, and it looks like a server that doesn't support EHLO should simply answer with an error message and continue the protocol, which is fine.

But I still don't like that this forces TLS on every connection without any settings options. I have seen misconfigured TLS in the past and I have no idea where python takes its root certificates from here or how to add custom ones. So this at least needs an option.

I think I would also prefer that option to be off by default to reduce configuration problems for users who don't quite know what their doing, unless you know a good reason why this should be otherwise.